experimental/dds/tree2/src/test/shared-tree/editing.spec.ts

```diff
@@ -290,6 +290,74 @@ describe("Editing", () => {
 			expectJsonTree(tree2, ["B", "A", "C"]);
 		});
 
+		it("can concurrently change node's value and move node", () => {
+			const tree1 = makeTreeFromJson(["A", "B"]);
+			const tree2 = tree1.fork();
+
+			// Change value of B to C
+			tree1.editor.setValue(
+				{ parent: undefined, parentField: rootFieldKeySymbol, parentIndex: 1 },
+				"C",
+			);
+
+			// Move B before A.
+			tree2.editor.move(
+				{ parent: undefined, field: rootFieldKeySymbol },
+				1,
+				1,
+				{ parent: undefined, field: rootFieldKeySymbol },
+				0,
+			);
+
+			tree1.merge(tree2);
+			tree2.rebaseOnto(tree1);
```

Comment:
This tests the rebasing of the mover over the value change. Consider testing the other way around as well. This could be done using an extra branch, but for legibility it might make sense to have 4 branches: one on which the setValue is done, one on which the move is done, one on which the two changes are merged in one order, and one on which the changes are merged in the other order.

Reply:
Same for the other test below.

Reply:
Thanks. It looks like one of these combinations may have found another bug.